### PR TITLE
VIH-5812 Delete audio application in close hearing

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-5812
### Change description ###

Delete audio recording application is added to close application action.

**Does this PR introduce a breaking change?** (check one with "x")

```
[X ] Yes
[ ] No
```

### CODE REVIEW

File: VideoAPI/Video.API/Controllers/ConferenceController.cs [hunk 1]
```diff
@@ -373,6 +374,17 @@ private async Task SafelyRemoveCourtRoomAsync(Guid conferenceId)
             }
         }
 
+        private async Task DeleteAudioRecordingApplication(Guid conferenceId)
+        {
+            var getConferenceByIdQuery = new GetConferenceByIdQuery(conferenceId);
```
Comment (RuwanikaWeerakkody) [inline VideoAPI/Video.API/Controllers/ConferenceController.cs:379]: Don't think we need this extra check here Valentina. We can just use the 404 returned form the delete instead.

Comment (RuwanikaWeerakkody) [reply VideoAPI/Video.API/Controllers/ConferenceController.cs:379]: We can remove this extra call

**Comment by valentinabennett** [comment]
We don't need to change Scheduler jobs, as function calls the api to close hearing and it have the added functionality to delete audio application.

File: VideoAPI/Video.API/Controllers/ConferenceController.cs (hunk 1 above)
Comment (valentinabennett) [reply VideoAPI/Video.API/Controllers/ConferenceController.cs:379]: I did not check here, I get HearingRefId to remove audio recording application, as it referenced to hearing Id, not conference Id

**Pull request merged**